packages/miniflare/src/plugins/core/errors/sourcemap.ts

```diff
@@ -2,6 +2,36 @@ import assert from "assert";
 import type { Options } from "@cspotcode/source-map-support";
 import { parseStack } from "./callsite";
 
+// `source-map-support` will only modify `Error.prepareStackTrace` if this is
+// the first time `install()` has been called. This is governed by shared data
+// stored using a well-known symbol on `globalThis`. To ensure...
+//
+// a) `miniflare` and `wrangler` can have differing `install()` options
+// b) We're not affecting external user's use of this package
+// c) `Error.prepareStackTrace` is always updated on `install()`
+//
+// ...load a fresh copy, by resetting then restoring the `require` cache, and
+// overriding `Symbol.for()` to return a unique symbol.
+export function getFreshSourceMapSupport(): typeof import("@cspotcode/source-map-support") {
+	const resolvedSupportPath = require.resolve("@cspotcode/source-map-support");
+
+	const originalSymbolFor = Symbol.for;
+	const originalSupport = require.cache[resolvedSupportPath];
+	try {
+		Symbol.for = (key) => {
+			// Make sure we only override the expected symbol. If we upgrade this
+			// package, and new symbols are used, this assertion will fail in tests.
+			assert.strictEqual(key, "source-map-support/sharedData");
```

Comment:
Should probably use an if-statement rather than an assertion to allow for other usage of `Symbol.for`